src/lib/cameras.js

```diff
@@ -118,5 +118,4 @@ function setOrthoCamera(camera, dir, ratio) {
   camera.bottom = info.bottom || -10;
   camera.position.copy(info.position);
   camera.rotation.copy(info.rotation);
-  camera.updateProjectionMatrix();
```

Comment:
This is now done in the updateAspectRatio() call that is done on cameratoggle event.